types/priv_validator.go

```diff
@@ -105,13 +97,14 @@ func LoadPrivValidator(filePath string) *PrivValidator {
 	if err != nil {
 		Exit(err.Error())
 	}
-	privVal := wire.ReadJSON(&PrivValidator{}, privValJSONBytes, &err).(*PrivValidator)
+	privVal := PrivValidator{}
```

Comment:
why not just `privVal := new(PrivValidator)`, since we always need the pointer ?